docs/articles/documentation/guides/basic-guides/obtain-client-side-info.md

```diff
@@ -164,6 +164,8 @@ test('Check client data', async t => {
 This approach only works for Node.js callbacks that fire during a test run. To ensure that the test function
 does not finish before the callback, suspend the test until the callback fires. You can introduce a promise and synchronously wait for it to complete, as shown in the example above.
 
+> The `boundTestRun` option requires the same test controller instance that is passed to the function used in a test declaration. It's unable to work with imported test controllers.
```

Comment:
It's a difficult to understand without examples.

Reply:
I think that adding code to a note is too much, but a link to the imported test controller docs could work well

Reply:
done